### ISSUE TITLE
High Contrast and Negative Contrast options remove background images

### ISSUE THREAD

**Issue by SplicedWeb** [pojome/one-click-accessibility#65]
It would improve this plugin if the background images could be treated as images.

Some design elements require background images to be used, but these are removed with the high contrast and negative contrast options so they do not show at all which can make the site content 'disappear' when its an image tile that flips on hover to shop text, there's no image to see to try to hover / click / touch.

**Comment by samuraiprojects** [comment]
You can fix this with CSS.

**Comment by SplicedWeb** [comment]
Yes, the point was more that this is no longer "one click accessibilty" at that stage...  It would be nice if your plugin didn't require themes to be rewritten / css overrides to be added.

**Comment by ivomarsan** [comment]
Same  here

**Comment by lukasj98** [comment]
> You can fix this with CSS.

How can I fix this with css ?
